Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -39,7 +39,10 @@ public static class Forms
 		public static bool IsInitialized { get; private set; }
 		static bool FlagsSet { get; set; }
 
-		internal static bool IsLollipopOrNewer
+		/// <summary>
```

Comment:
No need for this change

Reply:
There is no reason to keep this internal. It makes it easier to use the same logic in custom renderers as the built in ones.

Reply:
if it's public we have to maintain it and can't remove in the future, we would expose a public api but we haven't worked on it. I would prefer we keep this as is. 

Thanks

Reply:
Reverted this change back so the property is internal 👍